src/utils.jl

```diff
@@ -1,15 +1,23 @@
-using Distributions
-
-using Bijectors: Bijectors
 
+function pm_next!(pm, stats::NamedTuple)
+    ProgressMeter.next!(pm; showvalues=[tuple(s...) for s in pairs(stats)])
```

Comment:
Does this mean that all `stats` are required to be displayable in the progress meter? Seems like we could run into issues where someone wants some non-real stat or something where this would then just break.

Reply:
Hmm, that's actually a thinker.

Reply:
I'd suggest just listing some very basic stuff (objective value, gradient norm, etc.) by default, and then the user can add more if they want.

Reply:
I think plotting everything by default is okay. (Though we'll have to be more explicit about it then?) The user can store their own baggage themselves outside of the callback. Right?